src/rhenvision_changelog/changelog.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, repository: str, jira_url: str, jira_project: str, limit=None
         namespace, project = "/".join(split[3:-1]), split[-1]
 
         provider = GitHubJiraProvider(namespace, project, jira_url, jira_project)
-        super().__init__(repository, provider=provider, style="angular")
+        super().__init__(repository, provider=provider, convention="angular", parse_provider_refs=True)
```

Comment:
This was it ^^, they've changed the default here.

Reply:
Damn I saw this in the changelog but could not locate this bad boy.